cli/tools/repl/session.rs

```diff
@@ -291,7 +297,17 @@ impl ReplSession {
   ) -> Result<Value, AnyError> {
     self
       .worker
-      .with_event_loop(self.session.post_message(method, params).boxed_local())
+      .js_runtime
+      .with_event_loop(
+        self.session.post_message(method, params).boxed_local(),
+        PollEventLoopOptions {
+          wait_for_inspector: false,
+          // NOTE(bartlomieju): this is an important bit; we don't want to pump V8
+          // message loop here, so that GC won't run. Otherwise, the resulting
+          // object might be GC'ed before we have a chance to inspect it.
+          pump_v8_message_loop: false,
```

Comment:
This is the fix :)

Reply:
Wow. Thanks for noting it so heavily here.